app/models/twilio_notifier.rb

```diff
@@ -0,0 +1,29 @@
+require 'twilio-ruby'
+
+class TwilioNotifier
+  def initialize(order)
+    @order = order
+    
+  end
+
+  def ordered_item_titles
+    @order.items.pluck(:title).to_sentence
+  end
+
+  def notify
+    client.messages.create(
+      from: '+19495372518',
+      to: '+16082392471',
+      body: "Thank you for placing an order at Clamorous. Your #{ordered_item_titles} are being prepared by our expert chefs and will be delivered shortly. Your total is $#{@order.order_total_cost.round(2)}, our delivery team accepts all major credit cards."
+    )
+  end
+
+  private
+
+  def client 
+     @client ||= Twilio::REST::Client.new(ENV['twilio_account_sid'], ENV['twilio_auth_token'])
```

Comment:
Line is too long. [94/80]<br>Use 2 (not 3) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.